docs/admin/engines/settings.rst

```diff
@@ -429,15 +441,6 @@ Communication with search engines.
   Number of retry in case of an HTTP error.  On each retry, SearXNG uses an
   different proxy and source ip.
 
-``retry_on_http_error`` :
```

Comment:
Why removing this documentation? It still works per engine right?

Reply:
This documentation has been moved from the `outgoing:` section to the [`engines:` section](https://github.com/searxng/searxng/pull/2517/files/cc953f8efb6b219f4bdda3c7a908dbb9177dfbb3#diff-ac4f9c85d7e2e0f93a7c75f1b89a274590e65c98fa0c4f0b3da13daaa248efd9R636) (moved to line 636).